README.md

```diff
@@ -84,6 +84,9 @@ Versions 4.9.6 and below use outdated SSL certificates and are unsupported.
 
 ## Demo
 
+### Prerequisites
```

Comment:
Nit - this makes it read that the actual steps to run the demo are in "Prerequisities". Maybe install swiftlint can just be step 1 before pod install (similar comment in the other guide)